addons/content-index/src/main/java/org/commonjava/indy/content/index/ISPBinaryObjectKey2StringMapper.java

```diff
@@ -26,7 +26,11 @@
 import java.io.ObjectInputStream;
 import java.io.ObjectOutputStream;
 
-public class ISPKey2StringMapper
+/**
+ * @deprecated not used as this mapper will cause storage duplication because un-unique for equaled ISP object
+ */
+@Deprecated
```

Comment:
Was this mapper in place before this PR was started? If not, we can probably just remove the class, right?

Reply:
It is useless now as it will create duplicate record in db. I'll remove it.

Reply:
The other parts are ok. What left is the multi-build test to ensure this solution can work in a complex environment.

Reply:
:+1: